### ISSUE TITLE
Cache test failure on OS X 10.12

### ISSUE THREAD

**Issue by nguerrera** [dotnet/sdk#965]
Working on a mac tonight and hit these.

@ramarag @eerhardt 

```
/Users/nick/src/sdk/build/build.proj(168,5): error :    Microsoft.NET.Publish.Tests.GivenThatWeWantToCacheAProjectWithDependencies.compose_dependencies_noopt [FAIL]
/Users/nick/src/sdk/build/build.proj(168,5): error :       Following files cannot be found inside directory /Users/nick/src/sdk/bin/Debug/Tests/compose_dependencies_noopt/SimpleCache/outdir 
/Users/nick/src/sdk/build/build.proj(168,5): error :        runtime.osx.10.12-x64.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/osx.10.12-x64/native/libcoredistools.dylib
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/osx.10.12-x64/native/coredistools.h
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.runtime.native.system/4.4.0-beta-24821-02/runtimes/osx.10.12-x64/native/System.Native.a
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.runtime.native.system/4.4.0-beta-24821-02/runtimes/osx.10.12-x64/native/System.Native.dylib
/Users/nick/src/sdk/build/build.proj(168,5): error :       Stack Trace:
/Users/nick/src/sdk/build/build.proj(168,5): error :             at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
/Users/nick/src/sdk/build/build.proj(168,5): error :             at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
/Users/nick/src/sdk/build/build.proj(168,5): error :          /Users/nick/src/sdk/test/Microsoft.NET.TestFramework/Assertions/DirectoryInfoAssertions.cs(88,0): at Microsoft.NET.TestFramework.Assertions.DirectoryInfoAssertions.OnlyHaveFiles(IEnumerable`1 expectedFiles, SearchOption searchOption)
/Users/nick/src/sdk/build/build.proj(168,5): error :          /Users/nick/src/sdk/test/Microsoft.NET.Publish.Tests/GivenThatWeWantToCacheAProjectWithDependencies.cs(175,0): at Microsoft.NET.Publish.Tests.GivenThatWeWantToCacheAProjectWithDependencies.compose_dependencies_noopt()
/Users/nick/src/sdk/build/build.proj(168,5): error :    Microsoft.NET.Publish.Tests.GivenThatWeWantToCacheAProjectWithDependencies.compose_dependencies [FAIL]
/Users/nick/src/sdk/build/build.proj(168,5): error :       Following files cannot be found inside directory /Users/nick/src/sdk/bin/Debug/Tests/compose_dependencies/SimpleCache/outdir 
/Users/nick/src/sdk/build/build.proj(168,5): error :        runtime.osx.10.12-x64.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/osx.10.12-x64/native/libcoredistools.dylib
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/osx.10.12-x64/native/coredistools.h
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.runtime.native.system/4.4.0-beta-24821-02/runtimes/osx.10.12-x64/native/System.Native.a
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.runtime.native.system/4.4.0-beta-24821-02/runtimes/osx.10.12-x64/native/System.Native.dylib
/Users/nick/src/sdk/build/build.proj(168,5): error :       Stack Trace:
/Users/nick/src/sdk/build/build.proj(168,5): error :             at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
/Users/nick/src/sdk/build/build.proj(168,5): error :             at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
/Users/nick/src/sdk/build/build.proj(168,5): error :          /Users/nick/src/sdk/test/Microsoft.NET.TestFramework/Assertions/DirectoryInfoAssertions.cs(88,0): at Microsoft.NET.TestFramework.Assertions.DirectoryInfoAssertions.OnlyHaveFiles(IEnumerable`1 expectedFiles, SearchOption searchOption)
/Users/nick/src/sdk/build/build.proj(168,5): error :          /Users/nick/src/sdk/test/Microsoft.NET.Publish.Tests/GivenThatWeWantToCacheAProjectWithDependencies.cs(79,0): at Microsoft.NET.Publish.Tests.GivenThatWeWantToCacheAProjectWithDependencies.compose_dependencies()
/Users/nick/src/sdk/build/build.proj(168,5): error :    Microsoft.NET.Publish.Tests.GivenThatWeWantToCacheAProjectWithDependencies.compose_with_fxfiles [FAIL]
/Users/nick/src/sdk/build/build.proj(168,5): error :       Following files cannot be found inside directory /Users/nick/src/sdk/bin/Debug/Tests/compose_with_fxfiles/SimpleCache/outdir 
/Users/nick/src/sdk/build/build.proj(168,5): error :        runtime.osx.10.12-x64.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/osx.10.12-x64/native/libcoredistools.dylib
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/osx.10.12-x64/native/coredistools.h
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.runtime.native.system/4.4.0-beta-24821-02/runtimes/osx.10.12-x64/native/System.Native.a
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.runtime.native.system/4.4.0-beta-24821-02/runtimes/osx.10.12-x64/native/System.Native.dylib
/Users/nick/src/sdk/build/build.proj(168,5): error :       Stack Trace:
/Users/nick/src/sdk/build/build.proj(168,5): error :             at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
/Users/nick/src/sdk/build/build.proj(168,5): error :             at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
/Users/nick/src/sdk/build/build.proj(168,5): error :          /Users/nick/src/sdk/test/Microsoft.NET.TestFramework/Assertions/DirectoryInfoAssertions.cs(88,0): at Microsoft.NET.TestFramework.Assertions.DirectoryInfoAssertions.OnlyHaveFiles(IEnumerable`1 expectedFiles, SearchOption searchOption)
/Users/nick/src/sdk/build/build.proj(168,5): error :          /Users/nick/src/sdk/test/Microsoft.NET.Publish.Tests/GivenThatWeWantToCacheAProjectWithDependencies.cs(139,0): at Microsoft.NET.Publish.Tests.GivenThatWeWantToCacheAProjectWithDependencies.compose_with_fxfiles()
/Users/nick/src/sdk/build/build.proj(168,5): error :    Microsoft.NET.Publish.Tests.GivenThatWeWantToCacheAProjectWithDependencies.cache_nativeonlyassets [FAIL]
/Users/nick/src/sdk/build/build.proj(168,5): error :       Following files cannot be found inside directory /Users/nick/src/sdk/bin/Debug/Tests/cache_nativeonlyassets/UnmanagedCache/outdir 
/Users/nick/src/sdk/build/build.proj(168,5): error :        runtime.osx.10.12-x64.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/osx.10.12-x64/native/libcoredistools.dylib
/Users/nick/src/sdk/build/build.proj(168,5): error :       runtime.osx.10.12-x64.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/osx.10.12-x64/native/coredistools.h
/Users/nick/src/sdk/build/build.proj(168,5): error :       Stack Trace:
/Users/nick/src/sdk/build/build.proj(168,5): error :             at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message)
/Users/nick/src/sdk/build/build.proj(168,5): error :             at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args)
/Users/nick/src/sdk/build/build.proj(168,5): error :          /Users/nick/src/sdk/test/Microsoft.NET.TestFramework/Assertions/DirectoryInfoAssertions.cs(88,0): at Microsoft.NET.TestFramework.Assertions.DirectoryInfoAssertions.OnlyHaveFiles(IEnumerable`1 expectedFiles, SearchOption searchOption)
/Users/nick/src/sdk/build/build.proj(168,5): error :          /Users/nick/src/sdk/test/Microsoft.NET.Publish.Tests/GivenThatWeWantToCacheAProjectWithDependencies.cs(202,0): at Microsoft.NET.Publish.Tests.GivenThatWeWantToCacheAProjectWithDependencies.cache_nativeonlyassets()
/Users/nick/src/sdk/build/build.proj(176,5): error : There were test failures, for full log see /Users/nick/src/sdk/bin/Debug/Tests/TestResults.xml
```

**Comment by eerhardt** [comment]
I'll fix these today.

**Comment by nguerrera** [comment]
Thanks!

**Comment by eerhardt** [comment]
Fixed by #971
